index.js

```diff
@@ -432,7 +465,10 @@
     'String#fantasy-land/concat': function(s) { return this + s; },
 
     //  String#empty :: String ~> String
-    'String#fantasy-land/empty': function() { return ''; }
+    'String#fantasy-land/empty': function() { return ''; },
+
+    //  Undefined#equals :: Undefined ~> Undefined -> Boolean
+    'Undefined#fantasy-land/equals': function(other) { return true; }
```

Comment:
Why do we ignore `other`? what if it's not `undefined`? (same with `Null`)
looks like tests are correct but I don't understand why :D